src/cpp/shared/adventure/adv.h

```diff
@@ -228,6 +228,7 @@ class advManager : public baseManager {
   void PasswordEvent(mapCell *tile, hero *hero);
   int BarrierEvent(mapCell *tile, hero *hero);
 
+
```

Comment:
There are some blank lines in your commit....I'm guessing you mostly didn't mean them. git diff, git add -p, and git checkout -p are your friends.